docs/docs-beta/docs/guides/monitor-alert/custom-logging.md

```diff
@@ -3,4 +3,92 @@ title: "Setting up custom logging"
 sidebar_position: 20
 ---
 
-# Setting up custom logging
+
+# Custom loggers
+
+Custom loggers are used to alter the structure of the logs being produced by your Dagster pipelines. For example, JSON logs can be produced to more easily be processed by log management systems.
+
+
+## What you'll learn
+
+- How to write a custom logger
+- How to configure your jobs to use this custom logger
+
+<details>
+  <summary>Prerequisites</summary>
+
+To follow the steps in this guide, you'll need:
+
+- A basic understanding of Dagster concepts such as assets, jobs and definitions
+- A working knowledge of the Python logging module
+
+</details>
+
+
+## Step 1: Add a prebuilt custom logger to your jobs
+
+This step shows how to add an existing custom logger to your jobs.
+
+### Add the custom logger to your asset jobs
+
+The following example shows how to add the custom logger to your code location definitions and configure an asset job to use it.
+
+<CodeExample filePath="guides/monitor-alert/custom-logging/asset-job-example.py" language="python" title="Add custom logger to asset job" />
+
+
+### Add the custom logger to your task-based jobs
+
+Configuring a task job to use the custom logger slightly differs from the asset job example. The following example shows how:
+
+<CodeExample filePath="guides/monitor-alert/custom-logging/task-job-example.py" language="python" title="Add custom logger to task job" />
+
+
+### Changing the logger configuration in the Dagster UI
+
+You can also change the logger configuration in the Dagster UI. This is useful if you want to change the logger configuration without changing the code, to use the custom logger on an ad-hoc asset materialization, or change the verbosity of the logs.
+
+```yaml
+loggers:
+  console:
+    config:
+      log_level: DEBUG
+```
+
+## Step 2: Write your custom logger
+
+In this example, we'll create a logger implementation that produces comma separated values from selected fields in the
+log record. Other examples can be found in the codebase, in the built-in loggers such as `json_console_logger`.
+
+<CodeExample filePath="guides/monitor-alert/custom-logging/customlogger.py" language="python" title="Example custom logger" />
+
+Sample output:
+
+```csv
+2024-08-30T14:24:13.490405,dagster,INFO,3e092f8a-f2d7-4e34-8ff9-88b6c6a62255,demo_job,hello_logs,demo_job - 3e092f8a-f2d7-4e34-8ff9-88b6c6a62255 - hello_logs - Hello, world!
```

Comment:
nit: not a blocker, but maybe switch the format up to something simpler / not as cluttered (dropping the run id maybe?)